modules/utility.js

```diff
@@ -137,29 +137,18 @@ function checkPermission(msg, guildMember) {
     });
 }
 
-async function getUserFromID(msg, client) {
+async function getUserFromID(msg) {
+    const { args } = splitMessages(msg);
+    let user = msg.mentions.users.first();
```

Comment:
recomendation
in module
`function (args[0], msg.guild)`

```js
function (msg, guild)
let user = getUserFromMention(msg, guild.client);
if(!user && msg) user = msg;
if(!user) user = msg.author;
const member = guild.members.fetch(user)
```
fetch bisa di isi dengan user atau id
https://discord.js.org/#/docs/discord.js/stable/class/GuildMemberManager?scrollTo=fetch